content/02.main-text.md

```diff
@@ -432,7 +432,14 @@ Manubot provides an ideal platform for perpetual reviews [@arxiv:1502.01329; @ta
 
 Concepts for the future of scholarly publishing extend beyond collaborative writing [@doi:10.22541/au.148769949.92783646; @doi:10.22541/au.149693987.70506124; @tag:techblog-brown].
 Pandoc Scholar [@doi:10.7717/peerj-cs.112] and Bookdown [@doi:10.1201/9781315204963], which has been used for open writing [@doi:10.12688/f1000research.16516.1], both enhance traditional Markdown to better support publishing.
-The [knitcitations](https://github.com/cboettig/knitcitations) package enables citation by DOI or URL in R Markdown documents.
+[Dokieli](https://dokie.li/) stores articles as HTML and includes a clientside editor to allow reader annotation and interaction [@doi:10.1007/978-3-319-60131-1_33].
+[Texture](https://substance.io/texture/) is a browser-based editor for scholarly manuscripts stored as JATS.
+
+Several projects in addition to Manubot provide infrastructure for citation by identifier. 
```

Comment:
We use `citation by identifier` in some places and `citation-by-identifier` in others.  Is it worth standardizing?

Reply:
From https://www.grammarbook.com/punctuation/hyphens.asp

> Generally, hyphenate two or more words when they come before a noun they modify and act as a single idea. This is called a compound adjective

> When writing out new, original, or unusual compound nouns, writers should hyphenate whenever doing so avoids confusion.

I am leaning towards hyphenating everywhere since I think it's a bit more clear that citation-by-identifier is a distinct approach.

Reply:
I'm good with `citation-by-identifier`.